conda_env/specs/jinja2_file.py

```diff
@@ -0,0 +1,36 @@
+try:
+    import jinja2
+except ImportError:
+    jinja2 = None
+from .. import env
+from ..exceptions import EnvironmentFileNotFound, Jinja2NotInstalled
+
+
+class Jinja2Spec(object):
+    _environment = None
```

Comment:
I see.

Normally I'd just put that in `__init__`, defining as `None`.
Leaving it as a class variable will get you into trouble if this code is ran multiple times in the same process (certainly will happen if there are multiple tests for this)